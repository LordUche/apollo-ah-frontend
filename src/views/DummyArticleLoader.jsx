import React from 'react';
import ContentLoader from 'react-content-loader';

function DummyArticleLoader() {
  return (
    <div className="get-article__loader">
      <ContentLoader
        height={100}
        width={400}
        speed={2}
        primaryColor="#8b01be"
        secondaryColor="#ecebeb"
      >
        <rect x="0" y="0" rx="3" ry="3" width="70" height="10" />
        <rect x="80" y="0" rx="3" ry="3" width="100" height="10" />
        <rect x="190" y="0" rx="3" ry="3" width="10" height="10" />
        <rect x="15" y="20" rx="3" ry="3" width="130" height="10" />
        <rect x="155" y="20" rx="3" ry="3" width="130" height="10" />
        <rect x="15" y="40" rx="3" ry="3" width="90" height="10" />
        <rect x="115" y="40" rx="3" ry="3" width="60" height="10" />
        <rect x="185" y="40" rx="3" ry="3" width="60" height="10" />
        <rect x="0" y="60" rx="3" ry="3" width="30" height="10" />
      </ContentLoader>
    </div>
  );
}

export default DummyArticleLoader;
