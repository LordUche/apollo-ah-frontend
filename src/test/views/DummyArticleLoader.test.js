import React from 'react';
import setup from '../setup';
import DummyArticleLoader from '../../views/DummyArticleLoader';


describe('<NotFound />', () => {
  it('should render the component', () => {
    const wrapper = setup(<DummyArticleLoader />);
    expect(wrapper).toMatchSnapshot();
  });
});
