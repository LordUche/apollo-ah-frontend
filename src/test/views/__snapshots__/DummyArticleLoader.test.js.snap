// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render the component 1`] = `
<div
  className="get-article__loader"
>
  <ContentLoader
    height={100}
    primaryColor="#8b01be"
    secondaryColor="#ecebeb"
    speed={2}
    width={400}
  >
    <rect
      height="10"
      rx="3"
      ry="3"
      width="70"
      x="0"
      y="0"
    />
    <rect
      height="10"
      rx="3"
      ry="3"
      width="100"
      x="80"
      y="0"
    />
    <rect
      height="10"
      rx="3"
      ry="3"
      width="10"
      x="190"
      y="0"
    />
    <rect
      height="10"
      rx="3"
      ry="3"
      width="130"
      x="15"
      y="20"
    />
    <rect
      height="10"
      rx="3"
      ry="3"
      width="130"
      x="155"
      y="20"
    />
    <rect
      height="10"
      rx="3"
      ry="3"
      width="90"
      x="15"
      y="40"
    />
    <rect
      height="10"
      rx="3"
      ry="3"
      width="60"
      x="115"
      y="40"
    />
    <rect
      height="10"
      rx="3"
      ry="3"
      width="60"
      x="185"
      y="40"
    />
    <rect
      height="10"
      rx="3"
      ry="3"
      width="30"
      x="0"
      y="60"
    />
  </ContentLoader>
</div>
`;
